src/app/pages/meditationPlayer/MeditationPlayer.tsx

```diff
@@ -76,7 +77,7 @@ export default function MeditationPlayer({
         <Countdown>{`${h.toString().padStart(2, '0')}:${m
           .toString()
           .padStart(2, '0')}:${s.toString().padStart(2, '0')}`}</Countdown>
-        <div>{over ? 'Meditation stopped' : ''}</div>
+        {over ? <SuccessOverlay></SuccessOverlay> : ''}
```

Comment:
If it is empty, wouldn't it work as well as self-closing tag?
```suggestion
        {over ? <SuccessOverlay /> : ''}
```

Reply:
You are right! Thx!